test/unit/spriteManipulation.js

```diff
@@ -251,6 +251,55 @@ test('Sprite dance changes will throw with invalid parameters', async t => {
   t.end();
 });
 
+test('Sprite move sorting works reliably', async t => {
+  const moveNames = [
+    {
+      name: 'rest',
+      rest: true,
+    },
```

Comment:
Please update your test case to have a rest move further down the list, so we're actually checking that rest move sorting works.

Reply:
yes, we really need to have multiple cases. I'm fixing that.
The 1st priority was to make sure that we didn't change a properly sorted list, since that is the only case that we need today within the cdo repo.
Also, because of how levelbuilder is precomputing sort indexes, we really can't afford to resort these when this component is hosted within the cdo repo.
But that seems like a limitation upstream, and I'd like this component to behave in a rational way since the code made assumptions about where the different types of dance moves existed within the array (rests at the beginning, short bursts at the end)